pandas/tests/series/test_operators.py

```diff
@@ -1982,3 +1983,12 @@ def test_idxminmax_with_inf(self):
             assert np.isnan(s.idxmin(skipna=False))
             assert s.idxmax() == 0
             np.isnan(s.idxmax(skipna=False))
+
+    @pytest.mark.parametrize("num", [1.0, 1])
+    @pytest.mark.parametrize('op', [ops.add, ops.mul, ops.floordiv, ops.sub])
+    def test_series_bool_and_num_op_coerce(self, num, op):
+        # issue 18549
+        ser1 = pd.Series([op(num, num)])
+        ser2 = pd.Series([True])
+        assert_series_equal(ser1, op(ser2, num))
```

Comment:
use
``result = Series([op(num, num)])``
and
``assert_series_equal(result, expected)``
and write the expected out (and same in DataFrame tests).

Reply:
actually its hard to tell which is result and expected here.